store/details.go

```diff
@@ -41,6 +41,7 @@ type snapDetails struct {
 	Publisher        string             `json:"publisher,omitempty"`
 	RatingsAverage   float64            `json:"ratings_average,omitempty"`
 	Revision         int                `json:"revision"` // store revisions are ints starting at 1
+	ScreenshotURLs   []string           `json:"screenshot_urls,omitempty"`
```

Comment:
Yeah, let's please have some tests for this.

Also, what about the dimensions?  How will we ever get them filled if we only see URLs? It would be great to actually have the dimensions, at the same time it sounds a bit strange to put that API in place if we don't expect to ever see the data. Have we coordinated with the store folks to see if we can get some better info about the screenshots?